.githooks/install

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I meant was to remove the new githooks. I didn't know it already had ones. Please restore install and pre-commit and make sure there are no githook changes.
Just keep in mind that you need to keep out of modifications from the commit that is irrelevant to the topic of the PR which is new operations.

Reply:
resolve in commit [23eb758](https://github.com/moreh-dev/MIOpen/pull/15/commits/23eb7580715fb3ef2367cf2b9387671f68c3f7aa)